flink-clients/src/main/java/org/apache/flink/client/FlinkYarnSessionCli.java

```diff
@@ -146,6 +147,15 @@ public AbstractFlinkYarnClient createFlinkYarnClient(CommandLine cmd) {
 
 		flinkYarnClient.setConfigurationFilePath(confPath);
 
+		try {
+			FileSystem.setDefaultScheme(flinkConfiguration);
+		} catch (IOException e) {
+			LOG.error("Error while setting the default " +
```

Comment:
In the other CLI you rethrow the exception. I would also throw it here. In any case best to have it consistent imo.